src/test/java/unit/Group.java

```diff
@@ -315,6 +294,32 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private TestCommand.Result expectListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // call `terra group list-users` in table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list-users", "--name=" + name);
+
+    // assert header is correct
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("EMAIL", rowHead[0]);
+    assertEquals("POLICIES", rowHead[1]);
+
+    // assert email and policies are correct
+    boolean emailExists = false;
+    for (int i = 1; i < rows.length; i++) {
+      // Convert to lower-case because email in broad.json is mixed case
+      if (rows[i].contains(email.toLowerCase())) {
+        emailExists = true;
+        for (var policy : policies) {
+          assertThat("Row has all policies.", rows[i].contains(policy.toString()));
+        }
+      }
+    }
+    assertTrue(emailExists);
```

Comment:
Put a message like "Email address for group member should be included in table output." in this assertion.